src/main/java/org/jabref/gui/journals/ManageJournalAbbreviationsViewModel.java

```diff
@@ -27,14 +27,13 @@
 import org.jabref.logic.util.StandardFileType;
 import org.jabref.preferences.PreferencesService;
 
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Can you double check whether Google Guava supports that? In JabRef, we decided not to use Apache Commons Lang3 anymore, but only Google Guava. - We also have the Strings util class in JabRef. Maybe, this provides some methods.

Reply:
Probably he meant to use our StringUti class but the autocomplete suggested apache commons. Happened to me a couple of times ago

Reply:
I will add the same constant to our `StringUtil`, so `StringUtil.EMPTY` will be `""`.